packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts

```diff
@@ -182,32 +190,32 @@ const updateConfigOnEnvInit = async (context, category, service) => {
   return envParams;
 };
 
-const migrate = async context => {
+export const migrate = async (context: any): Promise<any> => {
   const category = 'auth';
   const { amplify } = context;
   const existingAuth = context.migrationInfo.amplifyMeta.auth || {};
-  if (!Object.keys(existingAuth).length > 0) {
+  if (!(Object.keys(existingAuth).length > 0)) {
     return;
   }
   const { provider, cfnFilename, defaultValuesFilename } = getSupportedServices().Cognito;
   const defaultValuesSrc = `${__dirname}/assets/${defaultValuesFilename}`;
 
-  const { roles } = require(defaultValuesSrc);
+  const { roles } = await import(defaultValuesSrc);
```

Comment:
The imported files are older JS. Please make sure this change doesn't break the `migrate` flow.

Reply:
Confirmed that this is working as expected with a migration from v6 -> 10.5.1